monkey/monkey_island/cc/services/mode/set_island_mode_service.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from monkey_island.cc.models.island_mode_model import IslandMode
+from monkey_island.cc.services.config_manipulator import update_config_on_mode_set
+from monkey_island.cc.services.mode.mode_enum import IslandModeEnum
+
+LOG = logging.getLogger(__name__)
+
+
+def set_mode(mode: IslandModeEnum):
+    island_mode_model = IslandMode()
+    island_mode_model.mode = mode.value
+    island_mode_model.save()
+    if not update_config_on_mode_set(mode):
+        LOG.error(
+            "Could not apply configuration changes per mode. Using default advanced configuration."
+        )
```

Comment:
This is basically the same as `monkey_island/cc/services/mode/island_mode_service.py::set_mode()`

Reply:
That file was split into `set_island_mode_service.py` and `get_island_mode_service.py` to avoid circular imports in f1a3e1fc25e8f018074490cf884c7e87318c6145.